scripts/devtron-reference-helm-charts/reference-chart_3-12-0/app-values.yaml

```diff
@@ -75,19 +75,22 @@ ReadinessProbe:
 
 ingress:
   enabled: false
-  annotations:
-    nginx.ingress.kubernetes.io/force-ssl-redirect: 'false'
-    nginx.ingress.kubernetes.io/ssl-redirect: 'false'
-    kubernetes.io/ingress.class: nginx
+  className: nginx
+  annotations: {}
+#    nginx.ingress.kubernetes.io/force-ssl-redirect: 'false'
+#    nginx.ingress.kubernetes.io/ssl-redirect: 'false'
+#    kubernetes.io/ingress.class: nginx
 #    nginx.ingress.kubernetes.io/rewrite-target: /$2
 #    nginx.ingress.kubernetes.io/canary: "true"
 #    nginx.ingress.kubernetes.io/canary-weight: "10"
 
   hosts:
     - host: chart-example1.local
```

Comment:
these had been commented out

Reply:
The chart automatically inserts kubernetes.io/ingress.class annotation by extracting the class from ingressClass field.
Also, since these values are the initial default values, removing any of these annotations will only affect fresh apps and not old apps.